### PR TITLE
W-11793337 Adding an option to the convert command

### PR DESCRIPTION
**Pull request by smotwani11**
@W-11793337@

This PR adds a definition file option to the convert command

### CODE REVIEW

**Review by peternhale — CHANGES_REQUESTED** [review]
@smotwani11 please have a look at the UT for package convert and add tests around definition file.

File: messages/package_convert.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ bypass the installation key requirement (either --installationkey or --installat
 
 Bypasses the installation key requirement. If you bypass this requirement, anyone can install your package. Either an --installationkey value or the --installationkeybypass flag is required.
 
+# definitionfile
+
+path to a definition file similar to scratch org definition file that contains the list of features and org preferences that the metadata of the package version depends on
```
Comment (jshackell-sfdc) [inline messages/package_convert.md:47]:
```suggestion
path to a definition file that contains features and org preferences that the metadata of the package version depends on
```

File: messages/package_convert.md [hunk 2]
```diff
@@ -42,6 +42,14 @@ bypass the installation key requirement (either --installationkey or --installat
...
+
+# longDefinitionfile
+
+The path to a definition file similar to scratch org definition file that contains the list of features and org preferences that the metadata of the package version depends on.
```
Comment (jshackell-sfdc) [inline messages/package_convert.md:51]:
```suggestion
The path to a definition file, similar to the scratch org definition file, that contains the list of features and org preferences that the metadata of the package version depends on.
```

Comment (smotwani11) [reply messages/package_convert.md:51]: I have updated the messages.

File: messages/package_convert.md (hunk 1 above)
Comment (smotwani11) [reply messages/package_convert.md]: Thanks. I have updated the messages.

**Review by peternhale — APPROVED** [review]

**Pull request merged**